MonoGame.Framework.Content.Pipeline/TargetPlatform.cs

```diff
@@ -42,6 +43,7 @@ public enum TargetPlatform
         /// Linux-based PCs
         /// (MonoGame)
         /// </summary>
+        [Obsolete("This platform is obsolete, use DesktopGL instead")]
         Linux,
```

Comment:
Ahh... yeah that would do it.  It was confused for a second because I was sure that worked.